ngraph/core/include/ngraph/op/loop.hpp

```diff
@@ -86,7 +86,12 @@ namespace ngraph
                 bool evaluate(const HostTensorVector& outputs,
                               const HostTensorVector& inputs) const override;
 
+            protected:
+                Loop(const Loop&);
```

Comment:
Why do you need to have this constructor?

Reply:
I have derived class `StaticShapeLoop` which just overrides `validate_and_infer_types` method. It's nice to have a way to create `StaticShapeLoop` instance from `Loop` - so I need copy constructor

Reply:
@ilyachur feel free to resolve if you're satisfied